### PR TITLE
KNES RO Configs

### PR DESCRIPTION
**Pull request by SovietOnion1608**
WIP, not the best. As of now, has Ariane 5, ATV, Hermes, and MRRC. Has a few issues as well, needs quite the work.

### CODE REVIEW

**Comment by Its-Just-Luci** [comment]
Ariane 5 SRB config is wrong. Line 29 in GameData/RealismOverhaul/RO_SuggestedMods/Knes/RO_Config_partial_Ariane.cfg needs to be `%engineType = EAP-241`

File: GameData/RealismOverhaul/RO_SuggestedMods/Knes/RO_Config_partial_ATV-STEAM.cfg [hunk 1]
```diff
@@ -0,0 +1,494 @@
...
+
+	@MODULE[ModuleReactionWheel]
+	{
+		@PitchTorque = 0.2
+		@YawTorque = 0.2
+		@RollTorque = 0.2
+	}
+}
+
+@PART[_Knes_ATV_Antenna]:FOR[RealismOverhaul]
```
Comment (StonesmileGit) [inline GameData/RealismOverhaul/RO_SuggestedMods/Knes/RO_Config_partial_ATV-STEAM.cfg:27]: Should this have a RA config?

File: GameData/RealismOverhaul/RO_SuggestedMods/Knes/RO_Config_partial_ATV-STEAM.cfg [hunk 2]
```diff
@@ -0,0 +1,494 @@
...
+	!RESOURCE[MonoPropellant]
+	{
+	}
+	@RESOURCE[ElectricCharge]
+	{
+		@amount = 2500
+		@maxAmount = 2500
+	}
+
+	MODULE
```
Comment (StonesmileGit) [inline GameData/RealismOverhaul/RO_SuggestedMods/Knes/RO_Config_partial_ATV-STEAM.cfg:59]: This module contains ROK related configs. SHould probably move there

File: GameData/RealismOverhaul/RO_SuggestedMods/Knes/RO_Config_partial_ATV-STEAM.cfg [hunk 3]
```diff
@@ -0,0 +1,494 @@
...
+	%RSSROConfig = true
+	@rescaleFactor = 1.8
+
+	@title = STEAM Pressurised Cargo Module A
+	@manufacturer = Aérospatiale
+	@description = A pressurized container, smaller than ATV Pressurized Cargo module, to contain cargo, delivering propellant, food, water, compressed gases and others, including dry cargo parts.
+
+	@mass = 1.02
+
+	MODULE
```
Comment (StonesmileGit) [inline GameData/RealismOverhaul/RO_SuggestedMods/Knes/RO_Config_partial_ATV-STEAM.cfg:281]: This module also contains ROK related configs

File: GameData/RealismOverhaul/RO_SuggestedMods/Knes/RO_Config_partial_ATV-STEAM.cfg [hunk 4]
```diff
@@ -0,0 +1,494 @@
...
+	%RSSROConfig = true
+	@rescaleFactor = 1.8
+
+	@title = STEAM Pressurised Cargo Module B
+	@manufacturer = Aérospatiale
+	@description = A pressurized container, smaller than ATV Pressurized Cargo module  and STEAM Pressurised Cargo Module A, to contain cargo, delivering propellant, food, water, compressed gases and others, including dry cargo parts.
+
+	@mass = 0.88
+
+	MODULE
```
Comment (StonesmileGit) [inline GameData/RealismOverhaul/RO_SuggestedMods/Knes/RO_Config_partial_ATV-STEAM.cfg:326]: Also has ROK related stuff

**Comment by Capkirk123** [comment]
Looked over this a little and fixed some minor issues. It's still pretty basic, but should be ok to merge.

**Pull request merged**